Sofa/framework/Core/src/sofa/core/objectmodel/Base.cpp

```diff
@@ -552,9 +552,13 @@ void  Base::parse ( BaseObjectDescription* arg )
         if(arg->getAttribute(attribute->m_name))
         {
             if(attribute->m_isRemoved)
-                msg_error() << "Attribute '" << attribute->m_name << "' has been removed. " << attribute->m_helptext;
+            {
+                msg_error() << "Attribute '" << attribute->m_name << "' has been removed in SOFA " << attribute->m_removalVersion << ". " << attribute->m_helptext;
```

Comment:
```suggestion
                msg_error() << "Attribute '" << attribute->m_name << "'is removed since SOFA " << attribute->m_removalVersion << ". " << attribute->m_helptext;
```

So it match the formulation in line 560.